src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1939,58 +1939,85 @@ public abstract partial class DbDataReader : System.MarshalByRefObject, System.C
+        public short GetInt16(string name) => throw null;
         public abstract int GetInt32(int ordinal);
+        public int GetInt32(string name) => throw null;
         public abstract long GetInt64(int ordinal);
+        public long GetInt64(string name) => throw null;
         public abstract string GetName(int ordinal);
         public abstract int GetOrdinal(string name);
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-        public virtual System.Type GetProviderSpecificFieldType(int ordinal) { throw null; }
+        public virtual System.Type GetProviderSpecificFieldType(int ordinal) => throw null;
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public System.Type GetProviderSpecificFieldType(string name) => throw null;
+        [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public virtual object GetProviderSpecificValue(int ordinal) => throw null;
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-        public virtual object GetProviderSpecificValue(int ordinal) { throw null; }
+        public object GetProviderSpecificValue(string name) => throw null;
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
-        public virtual int GetProviderSpecificValues(object[] values) { throw null; }
-        public virtual System.Data.DataTable GetSchemaTable() { throw null; }
-        public virtual System.IO.Stream GetStream(int ordinal) { throw null; }
+        public virtual int GetProviderSpecificValues(object[] values) => throw null;
+        public virtual System.Data.DataTable GetSchemaTable() => throw null;
+        public virtual System.IO.Stream GetStream(int ordinal) => throw null;
+        public System.IO.Stream GetStream(string name) => throw null;
         public abstract string GetString(int ordinal);
-        public virtual System.IO.TextReader GetTextReader(int ordinal) { throw null; }
+        public string GetString(string name) => throw null;
+        public virtual System.IO.TextReader GetTextReader(int ordinal) => throw null;
+        public System.IO.TextReader GetTextReader(string name) => throw null;
         public abstract object GetValue(int ordinal);
+        public object GetValue(string name) => throw null;
         public abstract int GetValues(object[] values);
         public abstract bool IsDBNull(int ordinal);
-        public System.Threading.Tasks.Task<bool> IsDBNullAsync(int ordinal) { throw null; }
-        public virtual System.Threading.Tasks.Task<bool> IsDBNullAsync(int ordinal, System.Threading.CancellationToken cancellationToken) { throw null; }
+        public bool IsDBNull(string name) => throw null;
+        public System.Threading.Tasks.Task<bool> IsDBNullAsync(int ordinal) => throw null;
```

Comment:
This has been removed from the implementation but not from here - please remove.

Reply:
Same as the previous similar comment. `public bool IsDBNull(string name)` is in the approved API.

Reply:
As above, the comment was about a new IsDBNullAsync overload which is no longer there.